src/rotest/core/result/handlers/stream/dots_handler.py

```diff
@@ -1,10 +1,30 @@
 """"Stream dots layout result handler."""
 # pylint: disable=invalid-name,too-few-public-methods,arguments-differ
 # pylint: disable=too-many-arguments,super-init-not-called,unused-argument
+from functools import wraps
+
 from base_handler import BaseStreamHandler
+from rotest.core.flow_component import AbstractFlowComponent
 from rotest.common.constants import GREEN, YELLOW, RED, BOLD, CYAN, BLUE
 
 
+def check_main_decorator(func):
```

Comment:
I don't like the usage of the word "decorator", I think it's redundant.
`def  ignore_blocks(...):` would be better because that's what it does, 
it ignores blocks and sub-flows. 
I think you need to figure out a name along those lines because otherwise the only way i'll know what this decorator is doing is by reading it's code.

Reply:
Agreed